panorama_connector.py

```diff
@@ -1675,14 +1717,15 @@ def _list_apps(self, param):
                 'key': self._key,
                 'xpath': APP_LIST_XPATH}
 
-        status = self._make_rest_call(data, action_result)
-
+        status, response = self._make_rest_call(data, action_result)
+        action_result.update_summary({'list_apps': response})
         if phantom.is_fail(status):
-            return action_result.set_status(phantom.APP_ERROR, PAN_ERR_MSG.format("retrieving list of application", action_result.get_message()))
+            return action_result.set_status(
+                phantom.APP_ERROR, PAN_ERR_MSG.format("retrieving list of application", action_result.get_message()))
 
         # Move things around, so that result data is an array of applications
         result_data = action_result.get_data()
-        result_data = result_data.pop(0)
+        result_data = result_data.pop()
```

Comment:
`pop()` is preferred because our goal is to interact with the response we've just received.

`pop(0)` can result in error, if action_result is modified prior to calling this function.